Santiago/challenge/todo-list/src/actions/actionCreators.js

```diff
@@ -0,0 +1,13 @@
+const taskAdded = () => {
```

Comment:
Review actions creators as we discussed at our meeting.

An action creator should return an action that goes to the dispatcher.

Reply:
I appreciate your feedback, I will take it into account.